components/layouts/navigation-tabs.tsx

```diff
@@ -60,54 +103,55 @@ export const NavigationTabs = ({ tab, children }: Props) => {
 					}}
 				>
 					{/* className="grid w-fit grid-cols-4 bg-transparent" */}
-					<div className="flex flex-row items-center justify-between  w-full gap-4 max-sm:overflow-x-auto">
-						<TabsList className="md:grid md:w-fit md:grid-cols-5  justify-start bg-transparent">
-							<TabsTrigger
-								value="projects"
-								className="rounded-none border-b-2 border-b-transparent data-[state=active]:border-b-2 data-[state=active]:border-b-border"
-							>
-								Projects
-							</TabsTrigger>
-							<TabsTrigger
-								value="monitoring"
-								className="rounded-none border-b-2 border-b-transparent data-[state=active]:border-b-2 data-[state=active]:border-b-border"
-							>
-								Monitoring
-							</TabsTrigger>
-
-							{(data?.rol === "admin" || user?.canAccessToTraefikFiles) && (
-								<TabsTrigger
-									value="traefik"
-									className="rounded-none border-b-2 border-b-transparent data-[state=active]:border-b-2 data-[state=active]:border-b-border"
-								>
-									Traefik File System
-								</TabsTrigger>
-							)}
-							{(data?.rol === "admin" || user?.canAccessToDocker) && (
-								<TabsTrigger
-									value="docker"
-									className="rounded-none border-b-2 border-b-transparent data-[state=active]:border-b-2 data-[state=active]:border-b-border"
-								>
-									Docker
-								</TabsTrigger>
-							)}
-
-							<TabsTrigger
-								value="settings"
-								className="rounded-none border-b-2 border-b-transparent data-[state=active]:border-b-2 data-[state=active]:border-b-border"
-							>
-								Settings
-							</TabsTrigger>
+					<div className="flex flex-row items-center justify-between w-full gap-4 max-sm:overflow-x-auto border-b border-b-divider pb-1">
+						<TabsList className="bg-transparent relative px-0">
+							{Object.keys(tabMap).map((key) => {
+								const tab = tabMap[key as TabState];
+								if (tab.isShow && !tab.isShow?.({ rol: data?.rol, user })) {
+									return null;
+								}
+								return (
+									<TabsTrigger
+										key={key}
+										value={key}
+										className="relative py-2.5 md:px-5 data-[state=active]:shadow-none data-[state=active]:bg-transparent"
+										onMouseEnter={() => setHover(key as TabState)}
+										onMouseLeave={() => setHover("")}
+									>
+										<span className="relative z-[1] w-full">
+											{tab.tabLabel || tab.label}
+										</span>
+										{key === activeTab && (
+											<motion.div
+												layoutId="indicator"
+												transition={{
+													duration: 0.25,
+												}}
+												className="absolute -bottom-[5.5px] w-full"
+											>
+												<div className="h-0.5 bg-foreground rounded-t-md" />
+											</motion.div>
+										)}
+										{key === hover && (
+											<motion.div
+												layoutId="bg"
+												transition={{
+													duration: 0.25,
+												}}
+												className="absolute w-full h-full"
+											>
```

Comment:
And i think the animation we want to achieve could something similar to this 

https://github.com/Dokploy/dokploy/assets/47042324/dcfbf056-16f3-4b86-a86b-1e76008ad63a

And mobile:

https://github.com/Dokploy/dokploy/assets/47042324/f009324e-fbe4-4ba1-ba2d-d4ce7d05173b